src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -275,105 +285,63 @@ For an example of how to build an internal integration, see [our Round Robin Iss
 
-#### How is the Sentry Integration different from OAuth Apps?
+Sentry integrations are similar to OAuth apps, except Sentry Integrations act as an independent entity. There is currently no way for Sentry integrations to act on behalf of a user.
 
-OAuth Applications allow an external developer to create an application which can authenticate as a Sentry user and take actions as the user within Sentry.
+### How is a Sentry integration different from personal auth tokens?
 
-Sentry Integration apps are similar to OAuth apps, except Sentry Integration apps act as an independent entity. There is currently no way for Sentry Integration apps to act on behalf of a user.
+Personal auth tokens are tokens a user can use to invoke APIs directly and have access to all the resources tied to that user. A Sentry integration doesn't represent a single user, but rather makes requests as "itself".
 
-#### How is the Sentry Integration different from personal Auth Tokens?
+### We use the Sentry API internally at our company. Should we switch to the Sentry Integration Platform?
 
-Personal Auth Tokens are tokens a user can use to invoke APIs directly and have access to all the resources tied to that user. Sentry Integration Apps do not represent a single user, but rather make requests as "itself."
-
-#### We use the Sentry API internally at our company. Should we switch to the Sentry Integration Platform?
-
-If you're looking for any of the new features below, we recommend you switch to the Sentry Integration Platform utilizing the [Internal Integrations](#internal-integrations).
+If you're looking for any of the new features below, we recommend you switch to Sentry Integrations and build an [internal integration](#internal-integrations).
 
 **Permissions**
+OAuth apps and auth tokens allow you to access Sentry as a specific user in your Sentry organization. This means:
 
-OAuth apps and Auth Tokens allow you to access Sentry as a specific user in your Sentry org. This means:
-
-- If this user leaves the org or their permissions change, the integration can break. A Sentry Integrations App, on the other hand, is a separate, independent entity with its own permissions.
-- OAuth apps and Auth Tokens permit the user to access other Sentry organizations they belong to, so there is limited isolation.
+- If this user leaves the organization or their permissions change, the integration can break. A Sentry integration, on the other hand, is a separate, independent entity with its own permissions.
+- OAuth apps and auth tokens permit the user to access other Sentry organizations they belong to, so there is limited isolation.
 - Actions taken in Sentry will connect to this user rather than the integration/app. For example, if your alerting integration automatically assigns issues to a teammate, it will appear in the history as “Alice assigned Bob issue XYZ” rather than “&lt;Your Sentry Integration apps&gt; assigned Bob issue XYZ.”
```

Comment:
```suggestion
- Actions taken in Sentry will connect to this user rather than the integration/app. For example, if your alerting integration automatically assigns issues to a teammate, it will appear in the history as “Alice assigned Bob issue XYZ” rather than “&lt;Your Sentry integration&gt; assigned Bob issue XYZ.”
```